pkg/operator2/secret.go

```diff
@@ -27,6 +28,17 @@ func (c *authOperator) expectedSessionSecret() (*corev1.Secret, error) {
 	return secret, nil
 }
 
+func (c *authOperator) bootstrapUserExists() bool {
+	b := bootstrapauthenticator.NewBootstrapUserDataGetter(c.secrets, c.namespaces)
+
+	enabled, err := b.IsEnabled()
```

Comment:
As this is a library call, I took the liberty of re-using it, rather can writing the same logic here.